lib/include/public/LogManagerBase.hpp

```diff
@@ -161,9 +177,16 @@ namespace MAT_NS_BEGIN
             return lock;
         }
 #endif
+        
+        static bool isGone()
+        {
+            return LogManagerExitHandler::isTerminated();            
```

Comment:
All of them share the same global static.. i.e. it has to run on any instance in case of multiple log managers, and so the flag trips.